backend/eventRoutes.py

```diff
@@ -52,37 +52,38 @@ def get(self):
 
 
 class Event(Resource):
-    def get(self, id):
-        return events.findEvent(id)
+    def get(self, eventId):
+        return events.findEvent(eventId)
 
     @requireAuthentication
-    def put(self, id):
+    def put(self, eventId):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/brownben/munro/pullRequest?prid=4764451)